src/element/sizeHelpers.test.ts

```diff
@@ -1,19 +1,29 @@
+import { vi } from "vitest";
 import { getPerfectElementSize } from "./sizeHelpers";
 import * as constants from "../constants";
 
 const EPSILON_DIGITS = 3;
-
+vi.mock(
+  "../constants.ts",
+  //@ts-ignore
+  async (importOriginal) => {
+    const module: any = await importOriginal();
+    return { ...module };
+  },
+);
```

Comment:
appears to do nothing, what's it for?

Reply:
This was needed to mock `constants.ts` file, in jest we can override by just assigning the value but in vite thats not possible hence I need to mock it so it allows me to override the value. I have added a comment regarding the same